modules/currency/currency.service.js

```diff
@@ -29,7 +32,7 @@ class CurrencyService {
   }
 
   async checkCurrencyExist(data) {
-    const currency = await Currency.find({
+    const currency = await Currency.countDocuments({
```

Comment:
Please, call variable properly, because it is not an actual entity, it is a count